packages/apps-config/src/links/index.ts

```diff
@@ -20,6 +21,8 @@ export const externalLinks: Record<string, ExternalDef> = {
   PolkassemblyIo,
   PolkassemblyNetwork,
   Polkastats,
+  SubId,
+  // eslint-disable-next-line sort-keys
```

Comment:
Please sort, the error is there for a reason :)

Reply:
Sorry. Done :)

Reply:
Sorry, I know (very, very, very well ... from experience) that these are a pain in the behind - however, as the numbers increase, it just adds some maintenance sanity.